api/services/elastigroup/aws/schemas/multimetric-metricconfig.yaml

```diff
@@ -0,0 +1,55 @@
+type: object
+title: Metric config
+required:
+  - name
+  - metricName
+  - namespace
+properties:
+  name:
+    type: string
+    description: |
+      The policy name
+    example: "policyName"
```

Comment:
Example should be a metric name, not policy name (better use the same example you use in the expression config - metric1)